### DISCUSSION TITLE
農用AI無人曳引機

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#280] [General]
老牌農業、林業機械製造商John Deere，在CES 2022上，發表了最新可大規模生產的無人曳引機。
若將來無人曳引機真的廣泛運用到了農業上，那麼農民就不用再如此辛苦的犁田了，還可以在本來要犁田的時間做其他事，大大增加了農業生產力，造福社會。
[John Deere發表農用AI無人曳引機](https://www.ithome.com.tw/news/148725)

**Comment by JAIMEHUNG** [comment]
如何制定價格使得農民負擔得起、製造廠商也能同時獲利變得很重要，不過對於農民真的助益良多

**Comment by ghost** [comment]
想當然爾又會有群人說農民要被機器取代、要失業了XD

**Comment by wax75397** [comment]
台灣的農民很多都是小農，如果這樣的機器過於昂貴的話，我想應該不太會有人買。反而是中國那邊的大規模農田比較有市場。

**Comment by yt7l0** [comment]
台灣的農業比較偏向精緻農業，不是大範圍耕種，曳引機可能比較沒有市場

**Comment by YuHengChen54** [comment]
感覺美國那種用飛機灑種子的地方比較用的到，不然小小一塊地要花大錢買這個感覺不是很划得來

**Comment by chupochun** [comment]
像是這種運用在農業上的應蓋是用在大型的農業上，比較有經濟效益。

**Comment by pekoranoinu** [comment]
無人曳引機想必能讓農業更接近無人自動化。

**Comment by yuchi06274** [comment]
台灣農業高齡化嚴重，嚴重缺乏青壯年人力，因此無人曳引機確實可以幫助台灣農業成長。但或許只能應用在南部較大型的農田，對北部精緻農業、自耕農可能沒什麼幫助。如果能被改良適合台灣農業就好了。

**Comment by xkllkx** [comment]
感覺在台灣，這種大型農具對台灣農業的效益不高。或許無人機噴灑肥料對種植茶葉的農業有幫助，可以克服地形障礙

**Comment by sys511613** [comment]
之前在鄉下已經看到有人用無人機再噴藥了(或是施肥)只能說科技進步真的很快，就算這種大型機具可能不適合在台灣用，但之後一定會出現適合用在台灣的科技農具

**Comment by minniewei** [comment]
希望在未來相關的科技應用能不斷應用

**Comment by treeann** [comment]
ai真的太多用途了

**Comment by stan890425** [comment]
如果像台灣這種小面積的農田應該不太合

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by alexthebigpp** [comment]
看來人類失業的日子指日可待。

**Comment by Ben6uo** [comment]
那之後是不是有自動埋地雷的機械人?

**Comment by Jimmy911218** [comment]
這感覺會適用於美國那種大面積的農不太適合台灣

**Comment by Hchejxnc** [comment]
台灣的農地太小了，期待為來有更適合的農業機具

**Comment by Zoe06060606** [comment]
人工智慧在農業上的應用又多了新的一種！
我認為在台灣的農業上透過人工智慧來控管農作物的溫濕度比較適合一些
